Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
I like the idea of running it on a larger project, especially one with multiple packages. I normally directly use the make file commands in my travis file if possible: https://github.com/ruflin/Elastica/blob/master/.travis.yml#L30 The reason is that I want to be able to run everything also locally with the same commands if possible.